TheJumpCloud/support
# Sa 2726 get raw command from gh

## Issues
* [SA-2726](https://jumpcloud.atlassian.net/browse/sa-2726) - SA-2726 Get Raw Command From GH

## What does this solve?
Converts GH JC commands gallery command url `(github.com)` to raw `(raw.githubusercontent.com)` urls to prevent changes to GitHub html from breaking Import-JCCommand
## Is there anything particularly tricky?
n/a
## How should this be tested?
1. Import GH JC commands gallery: https://github.com/TheJumpCloud/support/tree/master/PowerShell/JumpCloud%20Commands%20Gallery and make sure the right values are imported in the JC console
2.  Import a JC command with a short url: https://git.io/jccg-Mac-GetLoggedInUsers
